src/utils/helper.js

```diff
@@ -283,7 +286,7 @@ export function formatDate(date, props) {
 			if (dateFormats[props.queryFormat]) {
 				return date.toString(dateFormats[props.queryFormat]);
 			}
-			return date;
+			return date.getTime();
```

Comment:
Why are we returning `date.getTime()` here instead of returning `date`?

Reply:
Because if we don't have `queryFormat` we are  having `epoch_millis` as default as per the docs thats why in default I did this.